tutorials/airports/Tutorial.md

```diff
@@ -299,7 +299,7 @@ Once we've generated outline paths for each state, plotting them is straightforw
 
 Here we add a definition for a `path` mark to the `marks` array. We use the `layout_path` property for the path outline, set the `fill` color to light grey, and set the `stroke` color to white.
 
-```!vega
+```!vega(num[0-3600] scale=1200, num[0-900] cx=450, num[0-560] cy=280)
```

Comment:
Should we creat a separate vega-component repo and develop there? I think
we should have a standalone (JSON?) spec for components. The tutorials repo
can then focus on parsing and instantiating -- and we could still
devise whatever markdown syntax we like and translate it to the "canonical"
format.

On Saturday, August 15, 2015, Arvind Satyanarayan <notifications@github.com>
wrote:

> In tutorials/airports/Tutorial.md
> <https://github.com/vega/vega-tutorials/pull/1#discussion_r37136394>:
>
> > @@ -299,7 +299,7 @@ Once we've generated outline paths for each state, plotting them is straightforw
> >
> >  Here we add a definition for a `path` mark to the `marks` array. We use the `layout_path` property for the path outline, set the `fill` color to light grey, and set the `stroke` color to white.
> >
> > -```!vega
> > +```!vega(num[0-3600] scale=1200, num[0-900] cx=450, num[0-560] cy=280)
>
> Yeah, this is one of the areas to further explore. I started with this
> because it emulates method signatures, but parsing it is a pain. And agreed
> on clunkiness, that's why I initially shied away from JSON here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-tutorials/pull/1/files#r37136394>.
>